pkg/workspace/paths.go

```diff
@@ -61,6 +61,10 @@ const (
 	WorkspaceFile = "workspace.json"
 	// CachedVersionFile is the name of the file we use to store when we last checked if the CLI was out of date
 	CachedVersionFile = ".cachedVersionInfo"
+
+	// PulumiBookkeepingLocationEnvVar is a path to the folder where '.pulumi' folder is stored.
+	// The path should not include '.pulumi' itself. It defaults to the user's home dir if not specified.
+	PulumiBookkeepingLocationEnvVar = "PULUMI_BOOKKEEPING_LOCATION"
```

Comment:
The name "bookkeeping" feels a bit unusual here for a public facing thing.  Should it be `PULUMI_HOME_DIR`? or `PULUMI_HOME`?

Reply:
Yes, I copied it from an internal var, but I do realize it's ugly.
`PULUMI_HOME` would sound like `~/.pulumi` to me, so including `.pulumi`. Is there a name to indicate that it's one level above?

Reply:
I feel like it shouldn't append `.pulumi`. If you set `PULUMI_HOME_DIR`, it would be the full path (e.g. `PULUMI_HOME=/workspace/pulumi pulumi up`).

See https://getcomposer.org/doc/03-cli.md#composer-home for an example.

Reply:
I also think it shouldn't add .plumi, just use PULUMI_HOME value as is

Reply:
I think this makes sense. I renamed to `PULUMI_HOME` and now it's assumed to be the full path, so no `.pulumi` is appended. The default is still `~/.pulumi`.